src/apps/advanced-search-v2/components/AdvancedSearchFilterGroup.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+import {
+  ComplexSearchFacetsEnum,
+  ComplexSearchFacetValue
+} from "../../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../../core/utils/text";
+import CheckBox from "../../../components/checkbox/Checkbox";
+import iconExpandMore from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";
+
+interface AdvancedSearchFilterGroupProps {
+  facetField: ComplexSearchFacetsEnum;
+  label: string;
+  selectedValues: string[];
+  selectedCount: number;
+  facetValues: ComplexSearchFacetValue[];
```

Comment:
I wonder why we need to pass selected count. I thought we could derive this from the selected values and the facets.